### PR TITLE
Update/show toast options

### PR DESCRIPTION
**Pull request by phoward38**
- Updated `.options` argument for `showToast` to use defaults
  - Adding an option to the argument overwrites the default
  - **Exception:** `timeOut` & `extendedTimeOut` are 0 when `keepVisible=TRUE`

- Note: Options are only applied to a toast once all other toasts are off the screen. Otherwise, the new toast has the same options applied as the toast on screen
  - This is a `toastr` issue: https://github.com/CodeSeven/toastr/issues/565

### CODE REVIEW

File: R/showToast.R [hunk 1]
```diff
@@ -25,26 +43,29 @@ showToast <- function(
...
   session = shiny::getDefaultReactiveDomain()
 ) {
-
+  
+  # Incorrect option
+  if (!all(names(.options) %in% names(default_options))) {
+    stop('Incorrect option supplied', call. = FALSE)
+  }
+  
+  if (length(.options) == 0) {
```
Comment (merlinoa) [inline R/showToast.R]: I think you can replace this entire if else with:

```
.options <- modifyList(default_options, .options)
```

Please test that the above works.

File: R/showToast.R [hunk 2]
```diff
@@ -1,5 +1,21 @@
 
-
+#' @noRd default_options
+default_options <- list(
```
Comment (merlinoa) [inline R/showToast.R]: I would change the name of this to `showToast_default_options`.  `default_options` is a little too generic.

**Review by merlinoa — CHANGES_REQUESTED** [review]
@phoward38 please respond to my comments and lmk if you have questions

File: R/showToast.R (hunk 2 above)
Comment (phoward38) [reply R/showToast.R]: 9980df064c237c4fb83b8d87de40b609b04894a3

File: R/showToast.R (hunk 1 above)
Comment (phoward38) [reply R/showToast.R]: Good to know. I knew there had to be a function that already did what I was doing, just couldn't find it. Thanks!

0a63e18300f81a98eb335b47353337de929f2046

**Comment by phoward38** [comment]
@merlinoa Issues resolved

**Comment by phoward38** [comment]
@merlinoa Adding new `toastr.js.map` & `toastr.min.js`. Contains quick fix for the different `positionClass` argument bug. Taken from PR https://github.com/CodeSeven/toastr/pull/635 that has yet to be merged.

**Pull request merged**